test/JDBC/expected/TestSpatialFun-vu-verify.out

```diff
@@ -0,0 +1,932 @@
+SELECT STEquals(@point1, @point2);
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+DECLARE @point2 geometry = geometry::Point(3.0, 4.0, 4326);
+SELECT @point1 . STEquals ( @point2 );
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+SELECT PointColumn1.STEquals(PointColumn2) AS Equals FROM YourTableTemp2;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+SELECT PointColumn1.STEquals(@point1) AS Equals FROM YourTableTemp2;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @point1 geometry = geometry::Point(1.0, 2.0, 4326);
+SELECT @point1.STEquals(PointColumn2) AS Equals FROM YourTableTemp2;
+go
+~~START~~
+bit
+0
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+DECLARE @isEqual BIT = 1;
+SELECT * FROM YourTableTemp WHERE PointColumn.STEquals(@referencePoint) = @isEqual;
+go
+~~START~~
+int#!#geometry
+~~END~~
+
+
+SELECT ID, PointColumn1.STEquals(PointColumn2) AS Equal_points FROM YourTableTemp2;
+go
+~~START~~
+int#!#bit
+1#!#0
+~~END~~
+
+
+SELECT * FROM TableATemp JOIN TableBTemp ON PointA.STEquals(TableBTemp.PointB) <= 1;
+go
+~~START~~
+int#!#geometry#!#int#!#geometry
+1#!#E6100000010C000000000000F03F0000000000000040#!#1#!#E6100000010C00000000000008400000000000001040
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+SELECT * FROM TableATemp JOIN TableBTemp ON @referencePoint.STEquals(TableBTemp.PointB) <= 1;
+go
+~~START~~
+int#!#geometry#!#int#!#geometry
+1#!#E6100000010C000000000000F03F0000000000000040#!#1#!#E6100000010C00000000000008400000000000001040
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+SELECT * FROM TableATemp JOIN TableBTemp ON PointA.STEquals(@referencePoint) <= 1;
+go
+~~START~~
+int#!#geometry#!#int#!#geometry
+1#!#E6100000010C000000000000F03F0000000000000040#!#1#!#E6100000010C00000000000008400000000000001040
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+SELECT * FROM YourTableTemp ORDER BY @referencePoint.STEquals(PointColumn);
+go
+~~START~~
+int#!#geometry
+1#!#E6100000010C00000000000008400000000000001040
+2#!#E6100000010C00000000000014400000000000001840
+~~END~~
+
+
+DECLARE @referencePoint geometry = geometry::Point(0.0, 0.0, 4326);
+SELECT * FROM YourTableTemp ORDER BY PointColumn.STEquals(@referencePoint);
+go
+~~START~~
+int#!#geometry
+1#!#E6100000010C00000000000008400000000000001040
+2#!#E6100000010C00000000000014400000000000001840
+~~END~~
+
+
+SELECT * FROM YourTableTemp2 ORDER BY PointColumn1.STEquals(PointColumn2);
+go
+~~START~~
+int#!#geometry#!#geometry
+1#!#E6100000010C00000000000008400000000000001040#!#E6100000010C00000000000010400000000000001440
+~~END~~
+
+
+DECLARE @isEqual BIT = 1;
+SELECT ID, PointColumn1.STEquals(PointColumn2) AS isEqual,
+CASE WHEN PointColumn1.STEquals(PointColumn2) = @isEqual THEN 'yes' ELSE 'no'
+END AS isEqual
```

Comment:
Do we have a test case where we are getting STEquals as `1` for same points? Let's add one if not.

Reply:
resolved added